dev/plugins/hu.elte.txtuml.export.cpp/src/hu/elte/txtuml/export/cpp/templates/structual/FunctionTemplates.java

```diff
@@ -13,37 +14,54 @@ public static String functionDecl(String functionName) {
 	}
 
 	public static String functionDecl(String functionName, List<String> params) {
-		return FunctionTemplates.functionDecl(ModifierNames.NoReturn, functionName, params);
+		return FunctionTemplates.functionDecl(ModifierNames.NoReturn, functionName, params, "", false);
 	}
 
 	public static String simpleFunctionDecl(String returnType, String functionName) {
 		return PrivateFunctionalTemplates.cppType(returnType) + " " + functionName + "()";
 	}
 
 	// TODO modifiers (static, etc) - not supported yet
```

Comment:
Indeed, got ninja removed in [`722c2d4`](https://github.com/ELTE-Soft/txtUML/commit/722c2d4262d76aa8585cfdf4283dc535f885d973#diff-a22413200eda9887ce00a39479e932d1L56).